Godeps/Godeps.json

```diff
@@ -20,7 +20,11 @@
 		},
 		{
 			"ImportPath": "github.com/matttproud/golang_protobuf_extensions/ext",
-			"Rev": "ba7d65ac66e9da93a714ca18f6d1bc7a0c09100c"
+			"Rev": "fc2b8d3a73c4867e51861bbdd5ae3c1f0869dd6a"
```

Comment:
Hmm... why is `ext` still referenced here? It should be gone for good, shouldn't it?